### PR TITLE
[Native Image] Lock `intezer-sdk` at version 1.19.9

### PR DESCRIPTION
**Pull request by samuelFain**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Content Pull Request
Related PR: link to the PR at demisto/content

## Related Issues
Related: [CIAC-10263](https://jira-dc.paloaltonetworks.com/browse/CIAC-10263)

## Description
Lock `intezer-sdk` at version `1.19.9`, for more details see the related issue.

Effected images:
- `py3-tools-ubi`
- `py3-tools`
- `py3-native`

### CODE REVIEW

File: docker/py3-tools/pyproject.toml [hunk 1]
```diff
@@ -8,7 +8,7 @@ readme = "README.md"
 [tool.poetry.dependencies]
 python = "~3.10"
 cloudshare = "*"
-intezer-sdk = "*"
+intezer-sdk = "<=1.19.9"
```
Comment (JudahSchwartz) [inline docker/py3-tools/pyproject.toml:11]: you will need to add to dependabot config to ignore this. please add link to the issue as a comment here

Comment (JudahSchwartz) [reply docker/py3-tools/pyproject.toml:11]: we sjould also lock native image

File: .github/dependabot.yml [hunk 2]
```diff
@@ -573,10 +573,14 @@ updates:
     directory: /docker/py3-tools
     schedule:
       interval: daily
+    ignore:
+      - dependency-name: "py3-tools-ubi"
```
Comment (dorschw) [inline .github/dependabot.yml:577]: This should be the python library :)

File: .github/dependabot.yml [hunk 3]
```diff
@@ -573,10 +573,14 @@ updates:
...
   - package-ecosystem: pip
     directory: /docker/py3-tools-ubi
     schedule:
       interval: daily
+    ignore:
+      - dependency-name: "py3-tools-ubi"
```
Comment (samuelFain) [inline .github/dependabot.yml:583]:
```suggestion
      - dependency-name: "intezer-sdk"
```

File: .github/dependabot.yml (hunk 2 above)
Comment (samuelFain) [inline .github/dependabot.yml:577]:
```suggestion
      - dependency-name: "intezer-sdk"
```

File: docker/py3-tools-ubi/pyproject.toml [hunk 4]
```diff
@@ -8,7 +8,7 @@ readme = "README.md"
 [tool.poetry.dependencies]
 python = "~3.10"
 cloudshare = "*"
-intezer-sdk = "*"
+intezer-sdk = "<=1.19.9"
```
Comment (dorschw) [inline docker/py3-tools-ubi/pyproject.toml:11]: using `^` promises we don't downgrade by mistake

```suggestion
intezer-sdk = "^1.19.9"
```

File: docker/py3-tools/pyproject.toml (hunk 1 above)
Comment (dorschw) [inline docker/py3-tools/pyproject.toml:11]:
```suggestion
intezer-sdk = "^1.19.9"
```

**Review by JudahSchwartz — APPROVED** [review]

**Pull request merged**